.github/workflows/covector-version-or-publish.yml

```
@@ -0,0 +1,47 @@
+name: version or publish
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  version-or-publish:
+    runs-on: ubuntu-latest
+    timeout-minutes: 65
+    outputs:
+      change: ${{ steps.covector.outputs.change }}
+      commandRan: ${{ steps.covector.outputs.commandRan }}
+      successfulPublish: ${{ steps.covector.outputs.successfulPublish }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # required for use of git history
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
```

```suggestion
          registry-url: 'https://registry.npmjs.org'
          cache: 'yarn'
```

we need this too then right?

---

Yeah, i think I didn't add it because the release workflow doesn't trigger that often and I didn't want to add the get-cache time to the PR creation thingy